test/rbs/test/runtime_test_test.rb

```diff
@@ -1,14 +1,13 @@
 require "test_helper"
-
 require "rbs/test"
 require "logger"
 
 return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7.0')
 
-class RBS::Test::RuntimeTestTest < Minitest::Test
+module TestSetupHelper
   include TestHelper
 
-  def test_runtime_test
+  def do_runtime_session(other_env: {})
```

Comment:
I think we can organize the test helpers as follows:

* `def run_runtime_test(other_env:)` to run the test and returns `[err, status]`
* `def assert_test_success(other_env = {})` which asserts `assert_operator status, :success?` and returns stderr output.
* `def assert_test_failure(other_env = {})` which asserts `refute_operator status, :success?` and returns stderr output.

Using the three helpers, we can define the tests like:

```
def test_runtime_test_success
  assert_test_success()
  assert_test_success("RBS_TEST_SAMPLE_SIZE" => "30")
  assert_test_success("RBS_TEST_SAMPLE_SIZE" => "ALL")
end

def test_runtime_test_error_with_mulformed_sample_size
  error_message = assert_test_failure("RBS_TEST_SAMPLE_SIZE" => "yes")
  assert_match /.../, error_message

  ...
end
```